src/config.ts

```diff
@@ -0,0 +1,19 @@
+export class AllOfUsConfig {
+
+  public workspaceNamespace: string;
+  public workspaceId: string;
+  public apiHost: string;
+  public cdrVersionBigqueryDataset: string;
+  public cdrVersionCloudProject: string;
+  public bucketName: string;
+
+  public setJson(jsonString: string): void {
```

Comment:
Make this static and rename to `fromJson()`? and treat this as immutable? I don't know if I'd expect this object to change out from under me

Reply:
Done.